src/main/java/levi9/UserService/exception/CustomException.java

```diff
@@ -0,0 +1,11 @@
+package levi9.UserService.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
+public class CustomException extends RuntimeException{
```

Comment:
Maybe rename `CustomException` to something meaningful. If nothing else, it can be `BadRequestException` for example, due to http status.